azure-quantum/tests/unit/test_qiskit.py

```diff
@@ -404,6 +406,47 @@ def test_plugins_estimate_cost_qiskit_ionq(self):
         # cost = backend.estimate_cost(circuit, shots=100e3)
         # self.assertEqual(np.round(cost.estimated_total), 240.0)
 
+    @pytest.mark.ionq
+    def test_plugins_estimate_cost_qiskit_ionq_passthrough(self):
+        circuit = self._3_qubit_ghz()
+        workspace = self.create_workspace()
+        provider = AzureQuantumProvider(workspace=workspace)
+        self.assertIn("azure-quantum-qiskit", provider._workspace.user_agent)
+        backend = provider.get_backend("ionq.simulator", is_passthrough=True)
+        cost = backend.estimate_cost(circuit, shots=100e3)
+        self.assertEqual(cost.estimated_total, 0.0)
+
+        backend = provider.get_backend("ionq.qpu", is_passthrough=True)
+        cost = backend.estimate_cost(circuit, shots=1024)
+        self.assertEqual(np.round(cost.estimated_total), 1.0)
+
+        backend = provider.get_backend("ionq.qpu", is_passthrough=True)
+        cost = backend.estimate_cost(circuit, shots=100e3)
+        self.assertEqual(np.round(cost.estimated_total), 66.0)
+
+        ## The following two tests are skipped until we can use a workspace
+        ## with this target available as part of the E2E tests.
+        # backend = provider.get_backend("ionq.qpu.aria-1")
+        # cost = backend.estimate_cost(circuit, shots=1024)
+        # self.assertEqual(np.round(cost.estimated_total), 1.0)
+
+        # backend = provider.get_backend("ionq.qpu.aria-1")
+        # cost = backend.estimate_cost(circuit, shots=100e3)
+        # self.assertEqual(np.round(cost.estimated_total), 240.0)
+
+    @pytest.mark.ionq
+    def test_get_ionq_backends(self):
+        workspace = self.create_workspace()
+        provider = AzureQuantumProvider(workspace=workspace)
+        self.assertIn("azure-quantum-qiskit", provider._workspace.user_agent)
+        provider.get_backend("ionq.simulator", is_passthrough=True)
+        provider.get_backend("ionq.simulator", is_passthrough=True, gateset="native")
+        provider.get_backend("ionq.simulator", input_data_format="ionq.circuit.v1", output_data_format="ionq.quantum-results.v1", gateset="qis")
+        provider.get_backend("ionq.simulator", is_passthrough=False)
+        provider.get_backend("ionq.simulator", is_passthrough=True, input_data_format="ionq.circuit.v1", output_data_format="ionq.quantum-results.v1", gateset="qis")
+        provider.get_backend("ionq.simulator")
+        provider.get_backend("ionq.simulator", input_data_format="qir.v1")
```

Comment:
Could you please do `self.assertIsInstance(backend, IonQSimulatorQirBackend)` for all of these backends to make sure the backend returned from `get_backend` is of expected type?